### DISCUSSION TITLE
Column Filter Support

### DISCUSSION THREAD

**Discussion by gjordanp** [glideapps/glide-data-grid#849] [Ideas]
Hi Guys, thanks for the amazing work on the data-grid.

I was wondering if is there any WIP on column filtering.

Regards

**Comment by jassmith** [comment]
What do you mean exactly?

**Reply by gjordanp** [reply]
I mean filter un colunnndata like excel does

![35615e05-8e47-45ff-91e8-010a7b5350a3.jpg](https://github.com/glideapps/glide-data-grid/assets/67670790/c00b5eee-dbaf-48cb-bc21-d195a38ac5a9)

Best regards

**Reply by jassmith** [reply]
This is outside the scope of the core package. Something like this could be added to the glide-data-grid-source package. Are you having trouble implementing such a filter?

**Reply by gjordanp** [reply]
Ok thanks, I haven't tried to do it, because of priorities in our app.
Let's hope it won't take me too much time.

Regards
